dashboard/application/app/services/request_tracker_service.rb

```diff
@@ -1,12 +1,25 @@
 class RequestTrackerService
 
+  attr_reader :selected_queue
+
+  def selected_queue=(queue)
+    # no-op on null string (form default)
+    return if queue==""
+    if @queues.include?(queue)
+      @selected_queue = queue
+    else
+      raise ArgumentError, "invalid queue selection"
+    end
+  end
+
   def initialize(config)
-    @queue = config[:queue_name]
+    @queues = config[:queues]
     @priority = config[:priority]
 
-    if !@queue || !@priority
-      raise ArgumentError, "queue_name and priority are required options for RT service"
+    if !@queues || !@priority
```

Comment:
Should be check for empty `queues` too?

Reply:
Done. 5518434d25660c7b283a14a0d6384e3385fbef4c